deploy/lvm-operator.yaml

```diff
@@ -343,16 +392,30 @@ spec:
                   description: Size specifies the total size of volume group.
                   pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
                   x-kubernetes-int-or-string: true
+                snapCount:
+                  description: SnapCount denotes number of snapshots in volume group.
+                  format: int32
+                  type: integer
                 uuid:
                   description: UUID denotes a unique identity of a lvm volume group.
                   minLength: 1
                   type: string
               required:
+              - allocationPolicy
               - free
               - lvCount
+              - maxLv
+              - maxPv
+              - metadataCount
+              - metadataFree
+              - metadataSize
+              - metadataUsedCount
+              - missingPvCount
               - name
+              - permissions
               - pvCount
               - size
+              - snapCount
               - uuid
               type: object
             type: array
```

Comment:
Why do we need the above changes in LVMNode CR?

Reply:
LVMNode CR makes use of VolumeGroup struct. Hence change done there reflects in the CR too.